site/about/glossary/glossary.qmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "ValidMind glossary"
```

Comment:
```suggestion
title: "Glossary"
```

This title feels like a fairer representation of what the glossary now contains — it's not just ValidMind stuff.